pkg/kubectl/cmd/patch/patch.go

```diff
@@ -73,6 +73,8 @@ var (
 	patchLong = templates.LongDesc(i18n.T(`
 		Update field(s) of a resource using strategic merge patch, a JSON merge patch, or a JSON patch.
 
+               It has different limitations for different resource and is hard to record in docs, please try by yourself.
```

Comment:
- Isn't this indentation a bit off?
- I am not sure if that statement adds any value. How does this help users? Wouldn't it be better if this was a link or so to RFC 7386 or RFC 6902 (or whatever is used under the hood)?

Reply:
For different resource, it has different limit indeed. refer to [comment](https://github.com/kubernetes/kubernetes/issues/74801#issuecomment-472253189)

Reply:
I agree the comment does not really help users. A more difficult, but more useful effort would be to sweep API field documentation and add comments on immutable fields, so that `kubectl explain` and API docs would indicate they were immutable